dts/bindings/ethernet/espressif,esp32-eth.yaml

```diff
@@ -21,3 +21,11 @@ properties:
       Interface (RMII) mode.
   phy-handle:
     required: true
+
+  ref_clk_gpio_output:
+    type: int
+    enum:
+      - 16
+      - 17
+    description: |
+      GPIO number to output RMII Clock.
```

Comment:
How about using standard phandle-array for gpios? e.g. `ref-clk-output-gpios = <&gpio0 16 0>;`

Reply:
I don't see benefits from it since this is not handled by the Zephyr GPIO API but the espressif HAL directly. phandle-array would not be also possible to validate by the YAML parser itself as it is done right now.

Reply:
This is about having single way to describe GPIOs in devicetree files, regardless how those are used by driver (and whether Zephyr GPIO API is used or not).

See `nordic,nrf-led-matrix.yaml` bindings, which uses standard gpios phandles for rows and columns, but internally in the driver just nRF specific representation of GPIOs are used.

Reply:
Fair enough, I've changed that into `phandle-array` with additional build asserts.